example/index.html

```diff
@@ -1,374 +1,383 @@
 <!DOCTYPE html>
 <html>
-	<head>
```

Comment:
This file still has a ton of changes - I'm not sure why any of them would be needed?

Reply:
This is super weird. I have removed most of my changes from the original and something is making it mark everything as changes. 

I will revert the file and put my changes